app/services/notifications/setup/initiate-session.service.js

```diff
@@ -9,22 +9,29 @@ const SessionModel = require('../../../models/session.model.js')
 
 const NOTIFICATION_TYPES = {
   invitations: {
-    prefix: 'RINV-',
-    type: 'Returns invitation',
     journey: 'invitations',
-    redirectPath: 'returns-period'
+    prefix: 'RINV-',
+    redirectPath: 'returns-period',
+    type: 'Returns invitation'
   },
   reminders: {
-    prefix: 'RREM-',
-    type: 'Returns reminder',
     journey: 'reminders',
-    redirectPath: 'returns-period'
+    prefix: 'RREM-',
+    redirectPath: 'returns-period',
+    type: 'Returns reminder'
   },
   'ad-hoc': {
-    prefix: 'ADHC-',
-    type: 'Ad hoc',
     journey: 'ad-hoc',
-    redirectPath: 'ad-hoc-licence'
+    prefix: 'ADHC-',
+    redirectPath: 'ad-hoc-licence',
+    type: 'Ad hoc'
+  },
+  'abstraction-alert': {
+    journey: 'abstraction-alert',
+    prefix: 'WAA-',
+    redirectPath: 'abstraction-alert',
+    subType: 'waterAbstractionAlerts',
```

Comment:
I have aded the subType and will update the others in a separate change. I can remove some unnecessary upstream logic.